app/views/templates/_form.html.erb

```diff
@@ -98,13 +98,15 @@
     <div class="tab-pane" id="help">
       <div>
         <h1><%= _('Template syntax') %></h1>
-        <%= _('Provisioning templates are generated using the Embedded Ruby (ERB) language.') %>
+        <%= _('%s are generated using the Embedded Ruby (ERB) language.') % @resource_class.model_name.human.pluralize %>
         <%= _('The following table and example snippet shows global methods, variables and class methods and how to use them.') %>
         <%= _('These are guaranteed to work via the safe mode rendering, to ensure a template can do nothing harmful.') %>
         <% unless Setting[:safemode_render] -%>
           <%= _('To enable safe mode, navigate to Settings page and enable Safemode rendering option.') %>
         <% end -%>
-        <%= link_to(_('For more info visit our documentation.'), documentation_url(controller.documentation_anchor)) if controller.respond_to?(:documentation_anchor) %>
+        <%= (_'For more info visit ') %>
+        <%= (link_to(_('our documentation'), documentation_url(controller.documentation_anchor)) + ' and ') if controller.respond_to?(:documentation_anchor) %>
+        <%= link_to(_('the DSL (Domain Specific Language) documentation.'), ApipieDSL.configuration.doc_base_url) %>
```

Comment:
Also, it's better to use `apipie_dsl_apipie_dsl_path` instead of `ApipieDSL.configuration.doc_base_url` as mentioned here: https://github.com/theforeman/foreman/blob/ce8190441d9cec2cb10d38b419724e488639a323/app/views/about/index.html.erb#L128

Reply:
Thanks. Fixed.